Pipfile.lock

```diff
@@ -1,378 +0,0 @@
-{
```

Comment:
You've also removed the `Pipfile.lock` file.

Reply:
Fixed!